<!-- Make sure launch ctrl_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />
    <node pkg="thewalkingdead" type="task6.py" name="task6" output="screen"/>
</launch>